### ISSUE TITLE
508-defect-2 [MOBILE]: BAM2 Debt Letters - Debt letter download must be legible at mobile sizes

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16760]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a mobile user, I want to be able to read the debt letter download page easily.

## Details

The debt letter download page has content that extends to the edge and spills over without being able to scroll. All content should be available without horizontal scrolling for readability. Screenshot attached below from iPhone 8 + mobile Safari.

## Acceptance Criteria

- [ ] Content is legible at mobile width (assume 320px or 400% zoom at 1280px width)

## Environment

* https://staging.va.gov/manage-va-debt/your-debt/debt-letters
* Test user 38

## WCAG Guidance
* [Understanding Success Criterion 1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

## Screenshots or Trace Logs

![79051](https://user-images.githubusercontent.com/934879/100799450-afd0e980-33ea-11eb-856d-383a57593bb8.jpg)

**Comment by bradl3yC** [comment]
@1Copenut Ready for validation

**Comment by 1Copenut** [comment]
@bradl3yC Looking 💯 Moving to close.

---
<img width="460" alt="Screen Shot 2020-12-07 at 5 06 26 PM" src="https://user-images.githubusercontent.com/934879/101416038-9b8b6180-38ae-11eb-8d3a-100349d3dc6b.png">
